mne/epochs.py

```diff
@@ -498,15 +505,35 @@ def plot(self, epoch_idx=None, picks=None, scalings=None,
             Whether to halt program execution until the figure is closed.
             Useful for rejecting bad trials on the fly by clicking on a
             sub plot.
+        n_epochs : int
+            The number of epochs to show on a continuous plot. Defaults to 20.
+        n_channels : int
+            The number of channels per view on a continuous plot.
+            Defaults to 10.
+        bad_color : Tuple
+            A matplotlib-compatible color to use for bad channels. Defaults to
+            (1.0, 0.0, 0.0) (red).
+        fig_title : str | None
+            The title of the window. If None, epochs name or <unknown> will be
+            displayed. Defaults to None.
+        kind : str
+            What kind of plot to draw. Possible values 'trellis', 'continuous'.
+            Defaults to 'trellis'.
 
         Returns
         -------
         fig : Instance of matplotlib.figure.Figure
             The figure.
         """
-        return plot_epochs(self, epoch_idx=epoch_idx, picks=picks,
-                           scalings=scalings, title_str=title_str,
-                           show=show, block=block)
+        if kind == 'trellis':
```

Comment:
actually, I don't like `continuous`. It's a very vague term to me. Any other suggestions?